scripts/apps/search/controllers/MultiActionBarController.tsx

```diff
@@ -115,27 +124,91 @@ export function MultiActionBarController(
     /**
      * Multiple item spike
      */
-    this.spikeItems = function() {
-        let message = gettext('Are you sure you want to spike the items?');
-        const assignedItems = _.get(privileges, 'privileges.planning') ?
-            multi.getItems().filter((item) => item.assignment_id) : [];
+    this.spikeItems = function(): void {
         const spikeMultiple = () => {
             spike.spikeMultiple(multi.getItems());
             $rootScope.$broadcast('item:spike');
             multi.reset();
         };
 
         if ($location.path() === '/workspace/personal') {
-            message = gettext('Do you want to delete the items permanently?');
-        } else if (assignedItems.length) {
-            message = gettext('Some item/s are linked to in-progress planning coverage, spike anyway?');
-        } else if (!_.get(config, 'confirm_spike', true)) {
-            spikeMultiple();
+            modal.confirm(gettext('Do you want to delete the items permanently?')).then(spikeMultiple);
             return;
         }
 
-        return modal.confirm(message)
-            .then(spikeMultiple);
+        const onSpikeMultipleMiddlewares
```

Comment:
From this line forwards, isn't it all duplicated code from `scripts/apps/archive/index.tsx`?

Reply:
[not duplicate](https://user-images.githubusercontent.com/2076953/59419505-5b3b4800-8dcb-11e9-9f33-85a4d3c86a91.png), but similar. I've moved it to a separate function.